packages/shared/src/components/cards/squad/SquadGrid.spec.tsx

```diff
@@ -48,9 +48,9 @@ const openedMembersModal = async () => {
   mockGraphQL(
     createSourceMembersMock(result, { id: defaultSquad.id, role: null }),
   );
-  const trigger = await screen.findByLabelText('Members list');
+  const trigger = await screen.findByTestId('squad-member-short-list');
```

Comment:
I've changed because before it was relying on [a incorrect aria-label](https://github.com/dailydotdev/apps/pull/3659/files#diff-da65813c8de94c39f6d693fa2eac4deb6b00444969d5d71ee0d1a167e1ba261aL52) and, once I changed it, a lot of tests were failing. Should I have to check another method for query this element?

Reply:
Can't you do it on this one then?
https://github.com/dailydotdev/apps/pull/3659/files#diff-da65813c8de94c39f6d693fa2eac4deb6b00444969d5d71ee0d1a167e1ba261aR49

Reply:
Done ✅